docs/api/gateway_client.rst

```diff
@@ -3,6 +3,6 @@ GatewayClient
 
 .. py:module:: starknet_py.net.gateway_client
 
-.. autoclass:: GatewayClient
+.. autoclass-with-examples:: GatewayClient
     :members:
```

Comment:
Is this needed if nothing is specified?

Reply:
Yes, it definitely is. Without that GatewayClient page contains only `__init__` method 😅